include/RAJA/pattern/atomic.hpp

```diff
@@ -303,7 +303,7 @@ class AtomicRef
 
   RAJA_INLINE
   RAJA_HOST_DEVICE
-  value_type *getPointer() const { return m_value_ptr; }
+  value_type volatile * getPointer() const { return m_value_ptr; }
```

Comment:
What's the reasoning behind this change?

Reply:
There was a compilation error regarding the return type. m_value_ptr is declared below as a value_type volatile *.

Reply:
good call